client/signup/steps/website-content/section-types/default-page-details.tsx

```diff
@@ -43,7 +43,7 @@ export function DefaultPageDetails( {
 
 	const onMediaUploadFailed = ( { mediaIndex }: MediaUploadData ) => {
 		dispatch(
-			imageUploadFailed( {
+			mediaUploadFailed( {
```

Comment:
Probably out of the scope of this PR, but I was wondering if there was any way to avoid these kinds of duplicated changes in the future. (contact-page-details.tsx & default-page-details.tsx)

Reply:
Not sure that is possible but one thing we can do is to bring the "media" data structure into one universal shape that is returned by the API.